README.md

```diff
@@ -3,7 +3,7 @@
 O3DE (Open 3D Engine) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
 
 ## Contribute
-For information about contributing to Open 3D Engine, visit https://o3de.org/docs/contributing/
+For information about contributing to Open 3D Engine, visit https://o3de.org/docs/contributing/.
```

Comment:
not sure why we don't make this a markdown URL Link

Reply:
It displays as a proper URL link on GitHub, so probably no one has bothered to fix it. Happy to update it if you'd prefer.

Reply:
nah, that's OK

Reply:
Updated.